### ISSUE TITLE
ERROR=1 & OUT=blank

### ISSUE THREAD

**Issue by christophered** [mittagessen/kraken#50]
Peace Be Upon you,
I have been training a new Arabic model for 32 hours reaching 269,000 epochs, it is noticed that starting from epoch 8000 the Error remains 1 and the OUT is empty (from 8000 to 269,000).

My training data are generated artificially, with these features:
- Arabic
- Contains no diacritics
- 300 dpi
- Times New Roman, regular, size 18
- I'm sure that the transcription is 100% correct, it is based on [tanzil.net/download](http://tanzil.net/download/) "simple clean, no puase marks, no signs"

Attached (click-on):
[The transcribed html file](https://github.com/mittagessen/kraken/files/1099301/out.zip)
[The extracted png/gt.txt files](https://github.com/mittagessen/kraken/files/1099307/output.zip)
[The training script train.sh](https://github.com/mittagessen/kraken/files/1099309/script.zip)
[The produced clstm models](https://github.com/mittagessen/kraken/files/1099319/clstm.mods.zip)
[The complete terminal log](https://github.com/mittagessen/kraken/files/1099321/clstm.269000.zip)

My training script:
```#!/bin/bash
set -x
set -a
sort -R manifest.txt > /tmp/manifest2.txt
sed 1,100d /tmp/manifest2.txt > train.txt
sed 100q /tmp/manifest2.txt > test.txt

report_every=1000
save_every=1000
maxtrain=2000000
target_height=48
dewarp=center
display_every=1000
test_every=1000
hidden=100
lrate=1e-4
save_name=arabic
'/home/bmwmy/Desktop/kra/clstm/clstmocrtrain' train.txt test.txt

**Comment by christophered** [comment]
```
*** charsep 
got 1998 files, 100 tests
got 38 classes
.stacked: 0.0001 0.9 in 0 48 out 0 38
.stacked.parallel: 0.0001 0.9 in 0 48 out 0 200
.stacked.parallel.lstm: 0.0001 0.9 in 0 48 out 0 100
.stacked.parallel.reversed: 0.0001 0.9 in 0 48 out 0 100
.stacked.parallel.reversed.lstm: 0.0001 0.9 in 0 48 out 0 100
.stacked.softmax: 0.0001 0.9 in 0 200 out 0 38
.
.
.
ERROR 8000 1     6321 6321
8000
TRU نونمٔوي
ALN 
OUT 
ERROR 9000 1     6321 6321
.
.
.
ERROR 268000 1     6321 6321
268000
TRU تثبل لاق تثبل مك لاق هثعب مث ماع ةٔيام هللا هتامٔاف
 ةٔيام تثبل لب لاق موي ضعب ؤا اموي
ALN 
OUT 
ERROR 269000 1     6321 6321
269000
TRU رفكي نٕاف ةوبنلاو مكحلاو باتكلا مهانيتٓا نيذلا كٔيلؤا
 اموق اهب انلكو دقف ءالٔوه اهب
ALN 
OUT

**Comment by christophered** [comment]
I found the solution, it seems to be a weird problem.
I removed these lines from the training script, and the problem was solved
``` 
sort -R manifest.txt > /tmp/manifest2.txt
sed 1,100d /tmp/manifest2.txt > train.txt
sed 100q /tmp/manifest2.txt > test.txt
```
Therefore, you can run these commands in the terminal, and after they finish executing close the current terminal, then you must open a new terminal and run the training script.
What a weird problem.
